src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+
+    public String getColor() {
+        return color;
+    }
+    public void setColor(String color) {
+        this.color = color;
+    }
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+    public Ball() {
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    private int number;
```

Comment:
The 'number' field should have an access modifier. In Java, it's recommended to use the private modifier for fields to adhere to the encapsulation principle.